camera/src/test.py

```diff
@@ -0,0 +1,181 @@
+
+    def submit(self, frame, labels):
+        timer_start = time.perf_counter()
+        for result in labels:
+            (lower_left, upper_right) = result["box"]
+            (box_x_min, box_y_min) = lower_left
+            name = result["name"]
+            score = result["score"]
+
+            cv2.rectangle(frame, lower_left, upper_right, (10, 255, 0), 2)
+
+            # Draw label
+            label = '%s: %d%%' % (name, score)
+            labelSize, baseLine = cv2.getTextSize(label, cv2.FONT_HERSHEY_SIMPLEX, 0.7, 2) # Get font size
+            label_ymin = max(box_y_min, labelSize[1] + 10)
+            cv2.rectangle(frame, (box_x_min, label_ymin-labelSize[1]-10), (box_x_min+labelSize[0], label_ymin+baseLine-10), (255, 255, 255), cv2.FILLED) # Draw white box to put label text in
+            cv2.putText(frame, label, (box_x_min, label_ymin-7), cv2.FONT_HERSHEY_SIMPLEX, 0.7, (0, 0, 0), 2) # Draw label text
+        
+        print("submit: " + str(time.perf_counter() - timer_start))
+        self.proc.stdin.write(frame.tobytes())
+
+class WebcamStream:
+    udpStreamEnabled = False
+    running = True
+    frame = None
+
+    def __init__(self, resolution=(1920, 1080), fps=30):
+        camera = cv2.VideoCapture(0)
+        if not camera.isOpened():
+            print("Error opening camera")
+            exit(-1)
+
+        (width, height) = resolution
+        camera.set(cv2.CAP_PROP_FOURCC, cv2.VideoWriter_fourcc("M", "J", "P", "G"))
+        camera.set(cv2.CAP_PROP_FRAME_WIDTH, width)
+        camera.set(cv2.CAP_PROP_FRAME_HEIGHT, height)
+        camera.set(cv2.CAP_PROP_FPS, fps)
+        self.camera = camera
+        (_, self.frame) = self.camera.read()
+        
+        Thread(target=self.update, args=()).start()
+    
+    def __del__(self):
+        self.running = False
+
+    def update(self):
+        while self.running:
+            # stream if wanted
+            (_, self.frame) = self.camera.read()
+
+            if self.udpStreamEnabled:
+                pass
+
+    def stop(self):
+        self.running = False
+
+    def getFrame(self):
+        return self.frame
+
+    def openStream(self):
+        pass
+
+    def closeStream(self):
+        pass
+
+stream = WebcamStream()
+output_stream = FFMPEGOutputStream()
+
+# Load map between model output and labels
+with open(LabelMapPath, 'r') as map:
+    labels = [line.strip() for line in map.readlines()]
+
+print (labels)
+
+# Load model
+tflite_interpreter = tflite.Interpreter(CheckpointPath, num_threads=4)
+tflite_interpreter.allocate_tensors()
+
+input_details = tflite_interpreter.get_input_details()
+output_details = tflite_interpreter.get_output_details()
+
+input_data_idx = input_details[0]["index"]
+output_boxes_idx = output_details[1]["index"]
+output_classes_idx = output_details[3]["index"]
+output_scores_idx = output_details[0]["index"]
+
+input_mean = 127.5
+input_std = 127.5
+
+results = []
+
+while True:
+    frame = stream.getFrame().copy()
```

Comment:
Is this where the object tracking is happening? May be a nit for me, but could you add some comments just saying where that stuff is happening because I think most of us aren't familiar with OpenCV and it would help me since I'm not keeping along with all of the development haha